src/java.base/share/man/java.1

```diff
@@ -4005,6 +4012,15 @@ Example:
 \f[CB]\-Xlog:gc:garbage\-collection.log\f[R]
 .RE
 .TP
+.B \f[CB]\-XX:+ExtendedDTraceProbes\f[R]
+\f[B]Linux and macOS:\f[R] Enables additional \f[CB]dtrace\f[R] tool probes
+that affect the performance.
+By default, this option is disabled and \f[CB]dtrace\f[R] performs only
+standard probes.
+Use the combination of these flags instead: -XX:+DTraceMethodProbes, -XX:+DTraceAllocProbes, -XX:+DTraceMonitorProbes
```

Comment:
The flags should be in a code font (use \`-XX:...\` in the markdown source).

Reply:
done